### ISSUE TITLE
Unhandled Exception: NoSuchMethodError: The method '<=' was called on null.

### ISSUE THREAD

**Issue by JTorrus** [DarshanGowda0/GeoFlutterFire#81]
Running this code
```dart
Stream<List<Data>> streamListByLocation(Position currentPosition) {
    final geoPoint =
        GeoFirePoint(currentPosition.latitude, currentPosition.longitude);

    Query ref = _db
        .collection("data")
        .where("isActive", isEqualTo: true);

    return _geo
        .collection(collectionRef: ref)
        .within(center: geoPoint, field: "point", radius: 150, strictMode: true)
        .map((docs) => docs.map((doc) => Data.fromFirestore(doc)).toList());
  }
```

Throws the following exceptions
> E/flutter (10103): #28     _BufferingStreamSubscription._sendData  (dart:async/stream_impl.dart:338:11)
> E/flutter (10103): #29     _BufferingStreamSubscription._add  (dart:async/stream_impl.dart:265:7)
> E/flutter (10103): #30     _ForwardingStreamSubscription._add  (dart:async/stream_pipe.dart:134:11)
> E/flutter (10103): #31     _MapStream._handleData  (dart:async/stream_pipe.dart:234:10)
> E/flutter (10103): #32     _ForwardingStreamSubscription._handleData  (dart:async/stream_pipe.dart:166:13)
> E/flutter (10103): #33     _rootRunUnary  (dart:async/zone.dart:1134:38)
> E/flutter (10103): #34     _CustomZone.runUnary  (dart:async/zone.dart:1031:19)
> E/flutter (10103): #35     _CustomZone.runUnaryGuarded  (dart:async/zone.dart:933:7)
> E/flutter (10103): #36     _BufferingStreamSubscription._sendData  (dart:async/stream_impl.dart:338:11)
> E/flutter (10103): #37     _DelayedData.perform  (dart:async/stream_impl.dart:593:14)
> E/flutter (10103): #38     _StreamImplEvents.handleNext  (dart:async/stream_impl.dart:709:11)
> E/flutter (10103): #39     _PendingEvents.schedule.<anonymous closure>  (dart:async/stream_impl.dart:669:7)
> E/flutter (10103): #40     _rootRun  (dart:async/zone.dart:1122:38)
> E/flutter (10103): #41     _CustomZone.run  (dart:async/zone.dart:1023:19)
> E/flutter (10103): #42     _CustomZone.runGuarded  (dart:async/zone.dart:925:7)
> E/flutter (10103): #43     _CustomZone.bindCallbackGuarded.<anonymous closure>  (dart:async/zone.dart:965:23)
> E/flutter (10103): #44     _rootRun  (dart:async/zone.dart:1126:13)
> E/flutter (10103): #45     _CustomZone.run  (dart:async/zone.dart:1023:19)
> E/flutter (10103): #46     _CustomZone.runGuarded  (dart:async/zone.dart:925:7)
> E/flutter (10103): #47     _CustomZone.bindCallbackGuarded.<anonymous closure>  (dart:async/zone.dart:965:23)
> E/flutter (10103): #48     _microtaskLoop  (dart:async/schedule_microtask.dart:43:21)
> E/flutter (10103): #49     _startMicrotaskLoop  (dart:async/schedule_microtask.dart:52:5)
> E/flutter (10103):

It's weird because it always worked until now, Versions => `0.13.1+1` for `cloud_firestore` and `2.0.3+8` for `geoflutterfire`.

Also, for additional information, here's my `flutter doctor -v` output

> [√] Flutter (Channel stable, v1.12.13+hotfix.7, on Microsoft Windows [Version 10.0.17763.1039], locale en-US)
>     • Flutter version 1.12.13+hotfix.7 at D:\workspace\devkits\flutter
>     • Framework revision 9f5ff2306b (2 weeks ago), 2020-01-26 22:38:26 -0800
>     • Engine revision a67792536c
>     • Dart version 2.7.0
> 
> [!] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
>     • Android SDK at D:\workspace\devkits\android
>     • Android NDK location not configured (optional; useful for native profiling support)
>     • Platform android-29, build-tools 29.0.3
>     • Java binary at: D:\Program Files\Android\Android Studio\jre\bin\java
>     • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b03)
>     X Android license status unknown.
>       Try re-installing or updating your Android SDK Manager.
>       See https://developer.android.com/studio/#downloads or visit https://flutter.dev/setup/#android-setup for detailed instructions.
> 
> [√] Android Studio (version 3.5)
>     • Android Studio at D:\Program Files\Android\Android Studio
>     • Flutter plugin version 43.0.1
>     • Dart plugin version 191.8593
>     • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b03)
> 
> [√] Connected device (2 available)
>     • SM G973F            • RF8M1387VHW   • android-arm64 • Android 10 (API 29)
>     • AOSP on IA Emulator • emulator-5554 • android-x86   • Android 9 (API 28) (emulator)
> 
> ! Doctor found issues in 1 category.

Thanks in advance!

**Comment by JTorrus** [comment]
Maybe #79 #80 related.

**Comment by DarshanGowda0** [comment]
Fixed by #80
